### ISSUE TITLE
In ResultHit setId and getId Do Not Mirror Each Other

### ISSUE THREAD

**Issue by gordonbanderson** [manticoresoftware/manticoresearch-php#17]
In `\Manticoresearch\ResultHit`

```
public function getId()
    {
        return $this->_data['_id'];
    }

    public function setId($id)
    {
        $this->_data['id'] = $id;
    }
```

Note that one has an index of `id` and the other `_id`

**Comment by githubmanticore** [comment]
fix #17 id in setId of ResultHit via commit https://github.com/manticoresoftware/manticoresearch-php/commit/55e145b579af4e3fc8d586fe41c1e071e3a8b8a7

**Comment by githubmanticore** [comment]
fix #17 id in setId of ResultHit via commit https://github.com/manticoresoftware/manticoresearch-php/commit/55e145b579af4e3fc8d586fe41c1e071e3a8b8a7
